coreneuron/io/lfp.hpp

```diff
@@ -91,15 +91,17 @@ struct LFPCalculator {
     template <typename Vector>
     void lfp(const Vector& membrane_current);
 
-    std::vector<double> lfp_values;
+    const std::vector<double> lfp_values() const noexcept {
```

Comment:
You should avoid copying the vector.
```suggestion
    const std::vector<double>& lfp_values() const noexcept {
```